lua/gp/config.lua

```diff
@@ -77,6 +77,9 @@ local config = {
 	-- curl_params = { "--proxy", "http://X.X.X.X:XXXX" }
 	curl_params = {},
 
+	-- log	file location
+	log_file = (os.getenv("TMPDIR") or os.getenv("TEMP") or "/tmp") .. "/gp.nvim.log",
```

Comment:
I would prefer `stdpath('state')` or `stdpath('cache')`. There is a bit of hesitation between both. I personnally think `cache` is ok, since I dont mind deleting XDG_CACHE_HOME. Some consider it as state. COmpared to /tmp, it survives reboots which is nice

Reply:
@teto if we go for stdpath dirs, why not `vim.fn.stdpath('log')`?

Reply:
```
stdpath({what})                                                *stdpath()* *E6100*
		Returns |standard-path| locations of various default files and
		directories.

		{what}       Type    Description ~
		cache        String  Cache directory: arbitrary temporary
		                     storage for plugins, etc.
		config       String  User configuration directory. |init.vim|
		                     is stored here.
		config_dirs  List    Other configuration directories.
		data         String  User data directory.
		data_dirs    List    Other data directories.
		log          String  Logs directory (for use by plugins too).
		run          String  Run directory: temporary, local storage
				     for sockets, named pipes, etc.
		state        String  Session state directory: storage for file
				     drafts, swap, undo, |shada|.

		Example: >vim
			echo stdpath("config")
```

Reply:
- https://github.com/search?q=%2Fvim.fn.stdpath..cache...*log%2F&type=code 920 results
- https://github.com/search?q=%2Fvim.fn.stdpath..log...*log%2F&type=code 180 results

cache it is

Reply:
> why not vim.fn.stdpath('log')?

I was looking at neovim's logging code and found about it. Seems logical to use stdpath('log') then